src/s2wasm.h

```diff
@@ -935,8 +981,13 @@ class S2WasmBuilder {
       } else if (match("end_block")) {
         bstack.back()->cast<Block>()->finalize();
         bstack.pop_back();
-      } else if (match(".LBB")) {
-        s = strchr(s, '\n');
+      } else if (peek(".LBB")) {
+        // FIXME legacy tests: it can be leftover from "loop" or "block", but it can be a label too
```

Comment:
Cool, thanks @sunfishcode.

Ok, @yurydelendik, looks like the right approach to me in this pull. So aside from the comments I noted, lgtm.